TACC/Core-Styles
# feat(trumps): breadcrumbs (changes for core-styles.docs)

## Overview / Changes

- Clearly support four markups for breadcrumbs.
- Add global spacing for breadcrumbs.

<details><summary>the four markups</summary>

Clearly support four markups for breadcrumbs:
- **`nav.s-breadcrumbs ol`**
- **`[role="navigation"].s-breadcrumbs ol`**
- **`.s-breadcrumbs--has-ul ul`** (not recommended, but needed for https://github.com/TACC/TACC-Docs*)
- `ol.s-breadrumbs`

<sub>* Unless I change theme markup, but I do not want to, because doing so increases maintenance.</sub>

</details>

## Related

- [TUP-297](https://jira.tacc.utexas.edu/browse/TUP-297)
- required by https://github.com/TACC/TACC-Docs/pull/2

## Testing

0. Load https://github.com/TACC/TACC-Docs/pull/2 at ____.
1. Verify breadcrumb UI matches [design](https://xd.adobe.com/view/dabaec5a-cc31-4c5c-876e-56d6d8cc359e-8fe9/screen/62c317eb-f68a-4fd1-8e9b-aa0e7b15e06c/).

## UI

| before | after |
| - | - |
| ![before](https://user-images.githubusercontent.com/62723358/214125152-bb84f4c7-d102-49ce-98bf-74bdaec1656e.png) | ![after](https://user-images.githubusercontent.com/62723358/214125155-60d09b4d-8457-48a5-8b32-a6d181116749.png) |
